src/gui/tray/UserLine.qml

```diff
@@ -183,6 +211,15 @@ MenuItem {
                                 color: parent.parent.hovered ? Style.lightHover : "transparent"
                             }
                         }
+
+                        Accessible.role: Accessible.Button
+                        Accessible.name: qsTr("Remove Account")
+                        Accessible.description: qsTr("Permanently removes account from Nextcloud client configuration")

```

Comment:
Looks good! Detail cause it’s only accessible.name, but "Remove account" should have "account" not capitalized. :)